src/utils/animateTransition.ts

```diff
@@ -19,5 +19,16 @@ export default (
     console.log('animateTransition:', { id, enabledOnAndroid })
   }
 
-  LayoutAnimation.configureNext(config || LayoutAnimation.Presets.easeInEaseOut)
+  LayoutAnimation.configureNext(
```

Comment:
Curious why this is necessary?

Reply:
One of the top crashes we have has to do with removing view layers that dont exist and I think it comes from this layout animation. I found another person online that said this change made it not crash and in the app it looks good animation wise so figured we could try it